### DISCUSSION TITLE
Increase font size

### DISCUSSION THREAD

**Discussion by JoseCadenas** [quasarframework/quasar#9817] [CLI - Electron mode]
Hey Everyone,

I was curious, what is the best way to increase the font size across the entire application? I found a js solution but its doesnt increase the size of everything, nor do the elements account for the extra size.

[Stack Overflow](https://stackoverflow.com/a/43513385/15535372)
```
fontSize (increaseFactor) {
      let txt = document.getElementById('q-app');
      let style = window.getComputedStyle(txt, null).getPropertyValue('font-size');
      let currentSize = parseFloat(style);
      txt.style.fontSize = (currentSize + increaseFactor) + 'px';
    }
```

**Comment by ontwikkelfabriek** [comment]
@JoseCadenas 

You could look at these posts:
https://forum.quasar-framework.org/topic/3483/fluid-typography-solved
https://forum.quasar-framework.org/topic/6949/how-to-control-text-size-in-a-responsive-way

Here's a pending issue about this:
https://github.com/quasarframework/rfcs/issues/3
